pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 0.2.0+3
+version: 0.3.0+4
```

Comment:
to differentiate between `0.2.0 (3)` - I now need to bump `0.3.0 (4)` 🤔  while I think this is still cumbersome as we discussed, I think we can retrace the release version later when we have the CD pipeline in hands 🤔

Reply:
ok, so please update PR title as well